molecule/driver/openstackdriver.py

```diff
@@ -109,16 +141,18 @@ def up(self, no_provision=True):
                     flavor=self._openstack.get_flavor(instance['flavor']),
                     auto_ip=True,
                     wait=True,
-                    key_name=self.molecule.config.config['openstack'][
-                        'keypair'], security_groups=instance['security_groups']
+                    key_name=self._keypair_name,
```

Comment:
This is referenced as _keypair_name, but we have a method named `keypair_name` above?